hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFormatReader.java

```diff
@@ -115,9 +206,14 @@ private HoodieLogBlock createCorruptBlock() throws IOException {
     inputStream.seek(currentPos);
     log.info("Next available block in " + logFile + " starts at " + nextBlockOffset);
     int corruptedBlockSize = (int) (nextBlockOffset - currentPos);
-    byte[] content = new byte[corruptedBlockSize];
-    inputStream.readFully(content, 0, corruptedBlockSize);
-    return HoodieCorruptBlock.fromBytes(content, corruptedBlockSize, true);
+    if (ioIntensiveReaderSupport) {
```

Comment:
I do think this needs to be pushed into blocks still.. I can't think of any other way for us to decouple how blocks are physically laid out for each log version.. 
i.e parts after magic, length, type, version..